.github/workflows/haskell.yml

```diff
@@ -427,6 +427,13 @@ jobs:
           ~/.cabal/bin/gen-hie
         key: ${{ runner.os }}-cache-gen-hie
 
+    - name: Install Haskell
+      id: install-haskell
+      uses: input-output-hk/actions/haskell@latest
+      with:
+        ghc-version: 9.10.1
```

Comment:
I tried to extract this value in an environment variable and to use it both the ghc matrix and here, like thi:
ghc: ["8.10.7", "9.2.8", "9.6.6", "9.8.4", "$latest_ghc"]
and 
ghc-version: "$latest_ghc"  at this line. 

It works, but annooyingly, it's not resolved in the name of the job, so this happens: 
![2025-01-09_18-26](https://github.com/user-attachments/assets/be0e2e91-67dc-41e5-a95d-cf4466b11ea0)

Which is more annoying in my opinion that the repetition of the ghc version. 
I guess they don't even have to be the same, was just thinking that it would be good for this one to not be forgotten and left to rot here :laughing:

Reply:
> tried to extract this value in an environment variable and to use it both the ghc matrix and here

That's am interesting approach. We definitely don't need to do it here, but the technique itself could be useful in genral

> not be forgotten and left to rot here

It is ok for it to use older compiler, unless the newest version of the tool is incompatible with an older compiler. So, there is no inherent reason why a tool must be compiled with the newest compiler.